src/MigrateOriginalMessageCalls/MigrateOriginalMessageCallsCommand.php

```
@@ -0,0 +1,129 @@
+
+In the case of getOriginalResponse(), the call will be rewritten to
+getAttribute('originalRequest', {requestVariable}).
+
+In the case of getOriginalUri(), the call will be rewritten to
+getAttribute('originalUri', {requestVariable}->getUri()).
+
+If any getOriginalResponse() calls are detected, the script will present
+a warning indicating the file(s) and detail how to correct these
+manually.
+EOT;
+
+    const HELP_OPT_SRC = <<< 'EOT'
+Specify a path to PHP files to scan for
+getOriginalRequest/getOriginalUri/getOriginalResponse calls. If not
+specified, assumes src/ under the current working path.
+EOT;
+
+    const TEMPLATE_RESPONSE_DETAILS = <<< 'EOT'
+To correct these files, look for a request argument, and update the
+following calls ($response may be a different variable):
+
+    $response->getOriginalResponse()
+
+to:
+
+    $request->getAttribute('originalResponse', $response)
+
+($request may be a different variable)
+EOT;
+
+    /**
+     * @var string Path from which to resolve default src directory
+     */
+    public $projectDir = '.';
+
+    /**
+     * Configure the console command.
+     */
+    protected function configure()
+    {
+        $this->setDescription('Migrate getOriginal*() calls to request attributes.');
+        $this->setHelp(self::HELP);
+        $this->addOption('src', 's', InputOption::VALUE_REQUIRED, self::HELP_OPT_SRC);
+    }
+
+    /**
+     * Execute console command.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int Exit status
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $src = $this->getSrcDir($input);
+        } catch (ArgvException $e) {
+            $console = $this->getErrorConsole($output);
+            $console->writeln(sprintf(
+                '<error>Unable to determine src directory: %s</error>',
+                $e->getMessage()
+            ));
+            return 1;
```

Let exceptions bubble up. If the exception message is meaningful, no additional code is needed